apps/storybook/src/Pan.stories.tsx

```diff
@@ -45,17 +45,40 @@ Disabled.args = {
   disabled: true,
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const TwoComponents: Story<Record<string, never>> = (args) => {
```

Comment:
I have to set the `args` parameter explicitly to avoid seeing a warning in the UI.

Reply:
You could leave the control for the `modifierKey` of `PanLeft` to make use of `args` if you need to put it anyway :shrug: